app/__init__.py

```diff
@@ -8,4 +8,70 @@
 
 @app.route('/')
 def index():
-    return render_template('index.html', title="MLH Fellow", url=os.getenv("URL"))
+    return render_template('index.html',
+                           title="MLH Fellow",
+                           url=os.getenv("URL"))
+
+
+# Test route to see if my template is rendering as expected
+@app.route("/template")
+def template():
+
+    # **Shape of Data**
+    #
+    # experiences = [{
+    #    "name": string,
+    #    "role": string,
+    #    "date": string
+    # }, ...]
+    #
+    # hobbies = [{
+    #    "name": string,
+    #    "img": string
+    # }, ...]
+    #
+    # education = [{
+    #    "name": string,
+    #    "location": string
+    # }, ...]
+
+    return render_template(
+        "main.jinja",
+        title="Super Fellow!",
+        experiences=[{
+            "name": "Google",
+            "role": "Software Engineer",
+            "date": "XX 2021 - XX 2022"
+        }, {
+            "name": "Amazon",
+            "role": "Systems Development Engineer",
+            "date": "XX 2018 - XX 2019"
+        }, {
+            "name": "Meta",
+            "role": "Production Engineer",
+            "date": "XX 2017 - XX 2018"
+        }],
+        hobbies=[
+            {
+                "name":
```

Comment:
Interesting that the key and value are on different lines. Could put them on the same line for consistency.